index.js

```diff
@@ -43,17 +38,32 @@ module.exports = (robot, options, visit) => {
     });
   }
 
+  // Trigger the event
+  function trigger(event) {
+    robot.webhook.emit(event.event, event);
+  }
+
   function schedule(installation, repository) {
     // Wait a random delay to more evenly distribute requests
     const delay = options.delay ? options.interval * Math.random() : 0;
 
     robot.log.debug({repository, delay, interval: options.interval}, `Scheduling interval`)
 
     setTimeout(() => {
-      // Schedule visit to this repository on an interval
-      intervals[repository.id] = setInterval(() => visit(installation, repository), options.interval);
-      // Make the first visit now
-      visit(installation, repository);
+      const event = {
```

Comment:
for additional thoughts, if this could take advantage of `robot.auth` to create the GitHubAPI, then the current technique of mocking the `github` and `robot.auth` for unit testing purposes would also now extend to **probot-visitor/scheduler**